network-boot-lab.md

```diff
@@ -111,7 +112,7 @@ Modify `EOS_PUB_DEV_KEY` with the public key created in the [Key Management]() s
 > This can be changed later on using the privileged `set_param` function. Read more [here](https://guide.eoscostarica.io/docs/eos-learn/important-functions#set_params).
 
 ## Start the nodeos Service
-Before starting the nodeos service, it is necessary to create the file `genesis_start.sh`. For this, execute the following commands:
+Before starting the nodeos service, it is necessary to create the file `genesis_start.sh`. For this, execute the following commands: --> a bit repetitive
```

Comment:
Change  `For this, execute the following commands:` to `Run the following commands:`

@kuronosec what do you think?

Reply:
I'm going to apply my suggestion to avoid the repetitive phrase.